src/ServerError.js

```diff
@@ -0,0 +1,26 @@
+import Parse from 'parse';
+
+/**
+ * Constructs a new ServerError object with the given message and source.
+ *
+ * The message is an error that is intended for the server logs
+ * while the source message is intended for the server logs to aid a developer
+ *
+ * @class ServerError
+ * @constructor
+ * @param {ErrorCode} code the Parse.Error code
+ * @param {String} message will be sent to clients
+ * @param {String} serverErrorLog additional information
```

Comment:
nit: the comment here refers to `serverErrorLog` but the parameter and property name are both `serverLogError`.